crates/telio-wg/src/wg.rs

```diff
@@ -635,33 +674,6 @@ impl State {
                 .await
                 .map(|r| r.errno == 0)
                 .unwrap_or_default();
```

Comment:
In the original logic `push` dictates if it pushes uapi request or syncs everything. The current way will do the second part regardless which is change in behavior